File: app/src/main/java/com/defianttech/convertme/SingleUnit.java
package com.defianttech.convertme;

/*
 * Copyright (c) 2014-2016 Dmitry Brant
 */
public class SingleUnit {

    private final String name;
    public String getName() { return name; }
    
    private final double multiplier;
    public double getMultiplier() { return multiplier; }
    
    private final double offset;
    public double getOffset() { return offset; }

    private boolean enabled = true;
    public boolean isEnabled() { return enabled; }
    public void setEnabled(boolean enabled) { this.enabled = enabled; }

    public SingleUnit(String unitName, double unitMultiplier, double unitOffset) {
        name = unitName;
        multiplier = unitMultiplier;
        offset = unitOffset;
        this.enabled = enabled;
    }
}

File: app/src/main/java/com/defianttech/convertme/SingleUnit.kt
package com.defianttech.convertme

/*
 * Copyright (c) 2014-2018 Dmitry Brant
 */
class SingleUnit(val id: Int, val name: String, val multiplier: Double, val offset: Double) {
    var isEnabled = true

    override fun toString(): String {
        return name
    }
}
